### ISSUE TITLE
Add deleteDelegation Absinthe functionality using participant IDs

### ISSUE THREAD

**Issue by jinjagit** [liquidvotingio/api#153]
Following #143, we can create delegations using participant ids (for pre-existing participants) in much the same way as with participant emails. Despite permitting participant ids in the absinthe deleteDelegation mutation, however, we have no matching action(s).  
  
This development should also include the voting results updates (on delegation deletions) that are covered in #43.

**Comment by oliverbarnes** [comment]
Makes sense. I'd like to discuss having separate clauses for each scenario, though. And for the each of the error handling scenarios. 

I had another look at the delegation creation resolver, and now it feels like it's doing a lot there. Same for the args validator - feels like it could be broken into different clauses as well? Maybe we can pair on this

**Comment by jinjagit** [comment]
> Maybe we can pair on this

Sounds like a good plan. :+1:
